content/en/docs/reference/using-api/cel.md

```diff
@@ -182,6 +182,50 @@ To perform an authorization check for a service account:
 See the [Kubernetes Authz library](https://pkg.go.dev/k8s.io/apiserver/pkg/cel/library#Authz)
 godoc for more information.
 
+### Kubernetes quantity library
+
+Kubernetes 1.28 adds support for manipulating quantity strings (ex 1.5G, 512k, 20Mi)
```

Comment:
Im pretty sure this is not accurate. @jpbetz how should we phrase the available library functions for each version? I'm not sure even I have a good grasp on which library is available where or how it is determined

Reply:
You're right, it's not.

Since [quantity is introduced in version 1.28](https://github.com/kubernetes/kubernetes/blob/f0dcf0614036d8c3cd1c9f3b3cf8df4bb1d8e44e/staging/src/k8s.io/apiserver/pkg/cel/environment/base.go#L81) and we [require compatibility with 1.27 in the 1.28 release](https://github.com/kubernetes/kubernetes/blob/f0dcf0614036d8c3cd1c9f3b3cf8df4bb1d8e44e/staging/src/k8s.io/apiserver/pkg/cel/environment/base.go#L44), quantity won't be available to use until 1.29.

The reason we let the function "bake" for 1 release is so that if anyone does a rollback from 1.29 to 1.28 and has quantity in CEL expressions, nothing will break.

The consequence of this is that we should probably wait one release to merge this doc update.  Apologies I didn't remember to point this out before.